pkg/chartutil/create.go

```diff
@@ -570,20 +570,16 @@ func Create(name, dir string) (string, error) {
 		return "", err
 	}
 
-	path, err := filepath.Abs(dir)
+	cdir, err := filepath.Abs(dir)
 	if err != nil {
-		return path, err
+		return cdir, err
 	}
 
-	if fi, err := os.Stat(path); err != nil {
-		return path, err
+	parentPath := filepath.Dir(cdir)
```

Comment:
By doing this, you change the current behaviour of `Create()`, which is a [publicly available API endpoint](https://pkg.go.dev/helm.sh/helm/v3@v3.6.3/pkg/chartutil#Create). Changing its internal behaviour will silently break users consuming the public API.

Notice how you had to change every invocation of `chartutil.Create` in the tests in order to match its new calling signature.

In other words, if I previously expected

```
chartutil.Create("/foo")
```

To create a new chart in directory `/foo`, then the same should hold true until Helm 4.